src/ticketmap/types.ts

```diff
@@ -0,0 +1,53 @@
+export interface TicketGroup {
+    tevo_section_name: string
+    retail_price: number
+}
+
+export interface NormalizedTicketGroup {
+    section: string
+    zone?: string
+    price: number
+}
+
+export interface SectionZoneMapping {
+    [section: string]: {
+        sectionName: string
+        zone?: string
+    }
+}
+
+export interface Percentiles {
+    [key: string]: string
+}
+
+export interface DefaultProps {
+    isZoneDefault: boolean
+    showLegend: boolean
+    mapFontFamily?: string
+    selectedSections: string[]
+    ticketGroups: TicketGroup[]
+    sectionPercentiles: Percentiles
+    mapsDomain: string
+    onSelection(sections: string[]): void
+}
+  
+export interface Props extends Partial<DefaultProps> {
+    venueId: string
+    configurationId: string
+}
+  
+export interface State {
+    mapSvg?: SVGSVGElement
+    sectionZoneMapping: SectionZoneMapping
+    ticketGroups: TicketGroup[]
+    selectedSections: Set<string>
+    isZoneToggled: boolean
+    isDragging: boolean
+    currentHoveredZone?: string
+    currentHoveredSection?: string
+    tooltipActive: boolean
+    tooltipSectionName: string
+    tooltipZoneId: string
+    tooltipX: number
+    tooltipY: number
+}
```

Comment:
There is more than one component in this directory, so the purpose of `DefaultProps`, `Props` and `State` is ambiguous. I'd like to move them back to `ticketmap/index.js`